internal/provider/user_role_grant_resource_test.go

```diff
@@ -40,12 +40,9 @@ func TestAccRoleGrantResource(t *testing.T) {
 	t.Parallel()
 	clusterName := fmt.Sprintf("%s-rg-resource-%s", tfTestPrefix, GenerateRandomString(4))
 
-	// Skip the org test for the Acceptance Test because the test relies on the
-	// user having access to set an org level role. In order to avoid adding
-	// configuration, we use the service account user as the user to grant
-	// resources to.  This user doesn't seem to have the necessary permissions
-	// to perform the org grant so we skip it.
```

Comment:
I removed this logic because now we need the newly added service account user to have two roles so that we can check that on removal of one, the other one stays. So the user needs an org level and a cluster level role to test